library/src/main/java/com/schibsted/spain/barista/interaction/BaristaEditTextInteractions.kt

```diff
@@ -13,6 +14,11 @@ import org.hamcrest.Matchers.anyOf
 
 object BaristaEditTextInteractions {
 
+  /**
+   * This uses [androidx.test.espresso.action.ViewActions.replaceText],
+   * to remove any existing text and insert the characters directly,
+   * disregarding any EditText properties like `maxLength`, `textFilter`, etc.
```

Comment:
As a non-native English speaker, I would replace `disregarding` with `ignoring`. I think it means the same while it's easier to understand. But I could also study a bit more... Any thoughts there? Do you think this suggestion is valid?

Reply:
Yes I think it's good to change to the shorter word.